src/keystrokehandler.js

```diff
@@ -65,47 +67,48 @@ export default class KeystrokeHandler {
 	 * @param {Function} callback A function called with the
 	 * {@link module:engine/view/observer/keyobserver~KeyEventData key event data} object and
 	 * a helper to both `preventDefault` and `stopPropagation` of the event.
+	 * @param {Object} [options={}] Additional options.
+	 * @param {module:utils/priorities~PriorityString|Number} [options.priority='normal'] The priority of the keystroke
+	 * callback. The higher the priority value the sooner the callback will be executed. Keystrokes having the same priority
+	 * are called in the order they were added.
 	 */
-	set( keystroke, callback ) {
+	set( keystroke, callback, options = {} ) {
 		const keyCode = parseKeystroke( keystroke );
-		const callbacks = this._keystrokes.get( keyCode );
+		const priority = priorities.get( options.priority );
```

Comment:
Hmmm... it's been a while since I coded this and I don't remember. Try removing it and see if any test fails.